parachain/runtime/interlay/src/lib.rs

```diff
@@ -1078,6 +1078,24 @@ impl pallet_proxy::Config for Runtime {
     type AnnouncementDepositFactor = AnnouncementDepositFactor;
 }
 
+parameter_types! {
+    pub const ConfigDepositBase: Balance = 5 * DOLLARS;
+    pub const FriendDepositFactor: Balance = 50 * CENTS;
+    pub const MaxFriends: u16 = 9;
+    pub const RecoveryDeposit: Balance = 5 * DOLLARS;
+}
```

Comment:
How did you decide on these parameters?

Reply:
I had taken reference from the frame config set, but after re-reviewing, I suggest setting one of the following configs defined below.

INTR Config based on KSM Config set

Current Market Price
```
1 KSM -> 25 USD 
1 INTR -> 0.0159 USD 
```

```Rust
pub const ConfigDepositBase: Balance = 8 * Dollars; //0.0000000000125 USD

pub const FriendDepositFactor: Balance = 1 * Dollars; //0.00000000000125 USD

pub const MaxFriends: u16 = 9;

pub const RecoveryDeposit: Balance = 8 * Dollars; //0.0000000000125 USD
```


INTR Config based on Dollar, this 

```Rust
pub const ConfigDepositBase: Balance = 315 * Dollars; // if 1 INTR -> 0.0159 USD, this is equivalent to 5 Dollars

pub const FriendDepositFactor: Balance = 31 * Dollars; //this is equivalent to 0.5 Dollars

pub const MaxFriends: u16 = 9;

pub const RecoveryDeposit: Balance = 6300 * Dollars; // I recommend setting a larger value for the RecoveryDeposit compared to the ConfigDepositBase. 
// While many para chains set the RecoveryDeposit equal to the ConfigDepositBase, I suggest using a higher amount, 100 Dollars. 
// By increasing the RecoveryDeposit, we can ensure a more robust recovery process and mitigate the risk of unauthorized or malicious recoveries based on game theory.
```

Reply:
Note: Polkadot yet not have added the recovery pallet into the runtime.

Reply:
That makes sense to me!

Reply:
Aren't those values very very high? We require 250 vINTR for governance proposals and these would require a lot more INTR to actually use this feature. So only very few people could use this.

What's the downside of setting this to much lower values so that you could set this up with say max 500 INTR?

Reply:
@nud3l, I agree that we can reduce the value of `ConfigDepositBase` to 100 INTR to make it more accessible. However, it is important to keep the `RecoveryDeposit` at a sufficiently high level. This is because the `RecoveryDeposit` amount is refunded if the recovery process is successful. 

If we set the `RecoveryDeposit` value too low, there is a risk that a malicious rescuer could attempt to recover the account, and if the account holder cancels the recovery process, the malicious rescuer would only lose a minimal amount due to the low `RecoveryDeposit` value.

It's worth noting that the `RecoveryDeposit` only needs to be reserved if active account recovery is required and not when setting up the account recovery.